cli/src/program.rs

```diff
@@ -236,6 +242,23 @@ impl ProgramSubCommands for App<'_, '_> {
                                     "Use the designated program id even if the account already \
                                      holds a large balance of SOL",
                                 ),
+                        )
+                        .arg(
+                            Arg::with_name("compute_unit_price")
+                                .long("compute-unit-price")
+                                .takes_value(true)
+                                .help(
+                                    "Set computeUnitPrice (tx priotity fee) to boost transactions resulting in faster execution speed \
+                                    [Note: priority fee is added in LAMPORTS, so enter priority fee in terms of LAMPORTS]",
+                                ),
+                        ).
```

Comment:
we'll want to define this arg somewhere and such that it can be reused on all transaction-bearing subcommands, for consistencies sake. most likely that will be in the `clap-utils` dir, similarly to the `offline_signing` module

Reply:
can we do this in a diff pr ?